testsuite/smoke-test/src/txn_emitter.rs

```diff
@@ -79,11 +81,12 @@ async fn test_txn_emmitter() {
                 (TransactionType::default_call_different_modules(), 20),
                 (
                     TransactionType::CallCustomModules {
-                        entry_point: EntryPoints::BytesMakeOrChange {
-                            data_length: Some(32),
-                        },
+                        entry_point: EntryPoints::StepDst,
+                        // entry_point: EntryPoints::BytesMakeOrChange {
```

Comment:
Why not just delete this?

Reply:
this file is generally for easy testing, so having way to uncomment and comment seems fine